setup.py

```diff
@@ -38,10 +38,15 @@
         'Environment :: Web Environment',
         'Intended Audience :: Developers',
         'License :: OSI Approved :: Zope Public License',
+        'Programming Language :: Python',
```

Comment:
I believe we usually don't add this one, just the detailed "... Python :: X.Y".

(It doesn't hurt.)